qa/tasks/rhceph_ansible.py

```diff
@@ -1100,6 +1103,25 @@ def _create_rbd_pool(self):
             ],
             check_status=False)
 
+    def _execute_ceph_commands(self):
+        mon_node = self.ceph_first_mon
+        cmds = self.config.get('exec', list())
+        for cmd in cmds:
+            if 'sudo' not in cmd:
+                cmd = "sudo {}".format(cmd)
+            log.info('Executing CEPH command : {}'.format(cmd))
+            retries = 3
```

Comment:
instead of hard coding here, can you add this as configurable? we can have default as 3. what do you think.

Reply:
we can take this as enhancement.